### ISSUE TITLE
Help for utilisation

### ISSUE THREAD

**Issue by Guenito** [j-wyatt-a/N_m3u8DL-RE-GUI#1]
Hello,
Thank you for the GUI version of the new m3u8 software, it is much appreciated. Not knowing the language of the software perfectly I am a little bit lost in its use. I have entered a m3u8 link, the name of the desired file and the link of the page where the content is located. I launched the download but it remains blocked indefinitely on 0%.
Did I do the right thing?

Perhaps I was mistaken in the context of using this new software? Is it dedicated to downloading tracks from streaming services like the CLI version? What is the difference with this new RE version?
Thanks for your clarifications!

**Comment by j2wyatt** [comment]
hello，在下载 m3u8 视频的时候出现异常是很平常的事情，它可能是由各种因素导致的。一般来说使用 cli 能下载的视频，这个 gui 也是可以的。

具体的说，gui 的软件日志信息是可以在软件里查看的，它的位置在：打开软件 ---> 设置（齿轮按钮）---> 控制台日志

**Comment by Guenito** [comment]
Can you make a gif of a lambda use?
I just realised that the CLI version downloads the audio correctly but not the subtitles, they remain in several .ts. It was working several days ago, maybe we need to wait for an update from nialoda but it seems to be focused on the new RE version.

**Comment by j2wyatt** [comment]
没有明白你说的 lambda gif 是指的什么

**Comment by Guenito** [comment]
Sorry, a video or a gif that shows the use of the software through an example. So that it is more understandable than a screen.

**Comment by j2wyatt** [comment]
你是说制作一个查看软件日志的演示视频是吧

**Comment by Guenito** [comment]
Yep, from the full use of the software, copying the links (no need to see their origins), and the manipulations to get to the final files.

**Comment by j2wyatt** [comment]
我懂了，更多的演示视频我还没有那么多的时间。
但是，现在我能给出查看日志的示意图
<img width="600" alt="point" src="https://user-images.githubusercontent.com/53614528/198877411-8a3896fe-590a-4ec3-86e2-0d7e0be4b7d0.png">

**Comment by Guenito** [comment]
At the moment the log remains empty, it's been 30 minutes that the file is waiting at 0%. I don't know if I put the right links in the right place to start the download. Hence my request for a video explaining how it all works.

**Comment by j2wyatt** [comment]
如果你的日志显示为空的话，我猜你用的是 windows 版本吧，那个的内容展示确实有些不稳定，一般来说过几分钟就会有文字显示出来的，你可以截图给我看一个。

如果你下载的进度一直是 0% 的话，那么说明肯定是下载的问题，一般来讲对软件的操作不会出错的，我们只需要看看日志内容，判断下载异常的原因就可以了。

下面的是添加任务的示意图：
<img width="600" alt="ex" src="https://user-images.githubusercontent.com/53614528/198878412-48e55598-94a5-4e64-9b8b-119461ee9a77.png">

**Comment by Guenito** [comment]
Thanks for the capture, that's what I filled in, for the moment nothing works for a new video. It stays stuck and the log is empty.
I'm trying to upload a video from D+ so maybe that explains why?

**Comment by j2wyatt** [comment]
没有那么复杂，如果 cli 是可用的，那么 gui 也是可用的，cli 不能用，gui 也不能用，就这么简单。gui 里展示的日志和 cli 里 console print 是一致的。

这是正常工作的样子：

https://user-images.githubusercontent.com/53614528/198879365-06b33942-7b06-4296-bac0-051a4ee56a94.mp4

**Comment by gagmeng** [comment]
> 没有那么复杂，如果 cli 是可用的，那么 gui 也是可用的，cli 不能用，gui 也不能用，就这么简单。gui 里展示的日志和 cli 里 console print 是一致的。
> 
> 这是正常工作的样子：
> 
>  print.mp4

m3u8地址和网页地址区别在哪呢？请教

**Comment by j2wyatt** [comment]
m3u8 地址就是在网页里提取出来的视频地址，下载器就是根据这个地址下载视频片段的。
例如：https://baidu.com/video/index.m3u8

网页地址就指你在哪个网页里看的视频，它不影响视频的正常下载，只是一个附加的数据参数。
例如：https://baidu.com/video/hello.html

**Comment by EasyDragon** [comment]
点提交没反应，报这个错

已清理残留进程
sqlserver 启动
后台服务启动
后台启动成功
任务管理器启动了 ...
异常啦: The "path" argument must be of type string. Received null
后台继续进行
异常啦: The "path" argument must be of type string. Received null
后台继续进行
异常啦: The "path" argument must be of type string. Received null
后台继续进行
异常啦: The "path" argument must be of type string. Received null
后台继续进行
异常啦: The "path" argument must be of type string. Received null
后台继续进行
异常啦: The "path" argument must be of type string. Received null
后台继续进行
